the-collab-lab/tcl-69-smart-shopping-list
# [KD] Add 'About' Page

## Description

This 'About' page provides information about the application and how to use it. The page is responsive for web and mobile devices.

Note: Currently there are temporary template images of phone screens that will be swapped out with screenshots of the application once the design is complete.

## Related Issue

Closes #53
Sub-issue of #14

## Acceptance Criteria

- [x] Add detailed, but concise steps explaining how to use the application.

## Type of Changes

Use one or more labels to help your team understand the nature of the change(s) you’re proposing. E.g., `bug fix` or `enhancement` are common ones.

## Updates

### Before

N/A

### After


https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/122697843/228f0b0a-d5b8-44fd-9fe1-1bfebe18b930


## Testing Steps / QA Criteria

Click on 'About' in the navbar to navigate to the explainer page. Scroll down to see steps for using the site. Try resizing your window to see that the contents are responsive.
